ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -8,6 +8,9 @@ build_root:
   project_image:
     dockerfile_path: Dockerfile.tools
 images:
+- dockerfile_path: Dockerfile.must-gather
+  from: os-min8
+  to: numaresources-operator-must-gather
```

Comment:
Let's call this `numaresources-must-gather` because the image is planned to possibly survive the eventual dismissal of the `numaresources-operator`